### PR TITLE
Feature/#144

### PR DESCRIPTION
**Pull request by dsteinberg**

### CODE REVIEW

File: tests/test_losses.py [hunk 1]
```diff
@@ -78,25 +80,27 @@ def test_categorical_likelihood(make_data):
 
 def test_sum_likelihood():
```
Comment (ltiao) [inline tests/test_losses.py]: These tests don't appear to check the error case where broadcasting occurs?

Comment (dsteinberg) [reply tests/test_losses.py]: Good point!

Comment (dsteinberg) [reply tests/test_losses.py]: That would be good, thanks man :-)

**Comment by dsteinberg** [comment]
Nice! thanks

**Pull request merged**